src/main/resources/openapi/schemas/hotel/Booking/Mail/ConfernceMailStructure.yaml

```diff
@@ -0,0 +1,41 @@
+type: object
+description: Email Structure Data Required
+properties:
+  firstName:
+    type: string
+    nullable: false
+    pattern: "[a-zA-Z]{3,20}"
```

Comment:
I think we should increase the maximum number of characters (some persons have names longer than 20 characters), We will not store the data so it is safe to increase it